INSTALL.rst

```diff
@@ -1,6 +1,244 @@
 Installing scikit-image
 =======================
 
+We outline 4 different ways which we recommend using to get started with image
+processing and scikit-image.
+
+1. `Scientific Python Distributions`_ -- Recommended
+2. `pip based installation`_
+3. `conda based installation`_
+4. `System-wide package manager`_
+
+After your installation, you should be able to run the following code
+from a Python console or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+Should you have any questions getting started using ``scikit-image`` after
+reading this guide, please open an issue on
+`github <https://github.com/scikit-image/scikit-image/issues>`_
+to get additional help.
+
+For those attempting to build scikit-image from source, or trying contribute
+code back to scikit-image, we've provided development instructions below.
+
+Scientific Python Distributions
+-------------------------------
+
+``scikit-image`` comes pre-installed with several Python distributions.
+These have the advantage of comming pre-installed with many other useful
+packages for scientific computing and image analysis. Follow the instructions
+on the respective websites to get going. If you don't have python installed on
+your computer, these are the fastest way to get started with your project.
+We can recommend the following 3 distributions:
+
+- `Anaconda <https://www.anaconda.com/distribution/>`_
+- `Python(x,y) <https://python-xy.github.io/>`_
+- `WinPython <https://winpython.github.io/>`_
+
+Note that the version of ``scikit-image`` installed with these distributions
+may different than the one you expect. This should not stop you from getting
+started with your project. We simply suggest you check the installed version of
+``scikit-image`` with the code snippet included at the top of this guide so
+that you can refer to the appropriate documentation.
+
+
+pip based installation
+----------------------
+
+To install ``scikit-image`` with `pip <https://pip.pypa.io/en/stable/>`_, you
+may use the following commands
+
+.. code-block:: sh
+
+  # Update pip to a more recent version
+  python3 -m pip install -U pip
+  # Install scikit-image
+  python3 -m pip install -U scikit-image
+
+
+This is the easiest way of obtaining the latest released version of
+``scikit-image`` if python is already available on your system.
+
+We also offer an easy way to install related packages. These are often included
+in scientific python distributions. To install many of them using ``pip`` use
+the command:
+
+.. code-block:: sh
+
+    python3 -m pip install scikit-image[optional]
```

Comment:
Cool! I didn't know we actually did this!

Reply:
Yeah, we have a few others, but I don't feel like telling everybody about them in a "beginner" tutorial
https://github.com/scikit-image/scikit-image/blob/master/setup.py#L130